packages/graph/package.json

```diff
@@ -70,7 +70,7 @@
         "cpx": "1.5.0",
         "d3-array": "1.2.4",
         "d3-drag": "1.2.4",
-        "d3-force": "1.2.1",
+        "d3-force": "2.1.1",
```

Comment:
This change will not work in IE11 (do we care?) - I suspect your changes will work with 1.2.1 as well?